internal/txlib/pull.go

```diff
@@ -366,6 +374,10 @@ func (task *FilePullTask) Run(send func(string), abort func()) {
 	sendMessage("Pulling file", false)
 
 	if languageCode == "" {
+		if args.Pseudo {
+			sendMessage("Pseudo doesn't make sense for source files, skipping", false)
+			return
+		}
```

Comment:
Maybe put this check on `cmd/main.go`?

Reply:
It does, however, when you say `tx pull -s -l fr` we say that it doesn't make sense to have source with languages at the same time and abort the operation. I think it's better to let them know that they are doing something wrong instead of letting them think that their flags are taken into consideration and then have them discover that the results are not what they originally expected. Does this make sense?

Reply:
Yes yes, makes sense. Moved the condition to main :)